bpf/bpf_lxc.c

```diff
@@ -117,7 +117,6 @@ static inline int ipv6_l3_from_lxc(struct __sk_buff *skb,
 				   __u32 *dstID)
```

Comment:
Typos in commit: 

"is not n endpoint identity or a host" --> "is not an endpoint identity or a host"
"The behavior was undefined to the most extend as cluster range detection is
   inaccurate." - not sure what this means.
"When the ipcache is first populated based on pod notifications, before the
   kvstore is available." - not a complete sentence.